src/tests/prepare.spec.js

```diff
@@ -142,6 +142,21 @@ describe('prepare', () => {
     );
     t.assert(doAsyncSideEffect.calledThrice, 'Should be called 3 times');
   });
+  it('Should support <React.Forwardref />', async () => {
+    const ParagraphWrapper = React.forwardRef((props, ref) => (
+      <p id="test" ref={ref}>{props.children}</p>
+    ));
+    const ref = React.createRef();
```

Comment:
You should verify that this ref is passed correctly throught the prepare. We are not that interesed in the resulting html, since we know that react does this correctly.

Reply:
Would testing that the ref is correct after mounting accomplish this?